test_finder.sh

```diff
@@ -12,6 +12,7 @@ CI_GROUP_PATTERN_PREFIX="OpenSearch-Dashboards-ci-group-"
 OSD_COMPONENT_TEST_MAP=( "OpenSearch-Dashboards:opensearch-dashboards"
                          "alertingDashboards:alerting-dashboards-plugin"
                          "anomalyDetectionDashboards:anomaly-detection-dashboards-plugin"
+                         "flowFrameworkDashboards:dashboards-flow-framework"
```

Comment:
Can you describe what the purpose of this change is?

Reply:
test_finder.sh file is used in github workflow below to get test specs.  https://github.com/saimedhi/opensearch-dashboards-functional-test/blob/main/.github/workflows/cypress-workflow-bundle-snapshot-based-ci-groups.yml